shard.yml

```diff
@@ -6,6 +6,10 @@ authors:
 
 crystal: 0.25.0
 
+targets:
+  lucky:
+    main: src/lucky.cr
+
```

Comment:
Good idea! I think this is fine in this PR since it is a separate commit. I'll rebase & merge so it doesn't get squashed